akka-actor/src/main/resources/reference.conf

```diff
@@ -276,6 +276,9 @@ akka {
 
         # Max number of threads to cap factor-based parallelism number to
         parallelism-max = 64
+
+        # Enable establishing local first-in-first-out scheduling mode for forked tasks that are never joined.
+        async-mode = on
```

Comment:
The comment is a bit confusing. Akka users might have not used FJP therefore "forked tasks that are never joined" does not help. Also the tasks that Akka schedules are never joined anyway.